cmd/faucet/faucet.go

```diff
@@ -443,9 +445,10 @@ func (f *faucet) apiHandler(w http.ResponseWriter, r *http.Request) {
 		}
 		// Retrieve the Ethereum address to fund, the requesting user and a profile picture
 		var (
-			username string
-			avatar   string
-			address  common.Address
+			unique_id string
```

Comment:
Please rename to either `id`, `uid`. `uniqueID` would also work but is a bit long and convoluted. Camel case is definitely not acceptable in a Go codebase though.

Reply:
Thanks for catching this. I mainly use rust these days so It has become a habit to use snake_case.

tbh I'm surprised that the linter didn't catch this.